repository/docker/Dockerfile.gcs-authz.ubuntu

```diff
@@ -44,18 +47,21 @@ COPY ./common ${BUILD_DIR}/common
 # All files should be owned by the datafed user
 RUN chown -R datafed:datafed ${DATAFED_DIR}
 
+RUN apt-get reinstall globus-connect-server54
```

Comment:
It can probably be removed, but some of the GCS containers I was using were not using the most up to date version. I'm not quite sure how I feel about this. I'm not sure how much effort is being put into ensuring the GCS containers provided by Globus are up to date.